control-plane/csi-driver/src/bin/node/node.rs

```diff
@@ -475,6 +475,7 @@ impl node_server::Node for Node {
         //===============================CsiAccessType=============================================
 
         let request = request.into_inner();
+        info!("Volume expansion request: {:?}", &request);
```

Comment:
Again, keep with existing ones..
Also might want to move this up?

```suggestion
        trace!("node_expand_volume {request:?}");
```

Reply:
Moved this up, keeping the request, instead of shadowing it.